src/components/AddTodo/AddTodo.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+import { TextField, Button, Box } from '@mui/material';
+import { makeStyles, createStyles } from '@mui/styles';
+import { useDispatch } from 'react-redux';
+import { addTodo } from '../../redux/todosSlice';
+
+const useStyles = makeStyles(() =>
+  createStyles({
+    addTodo: {
+      margin: '0 auto',
+    },
+  })
+);
+
+const AddTodo: React.FC = () => {
+  const [todoText, setTodoText] = useState('');
+  const dispatch = useDispatch();
+  const classes = useStyles();
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    if (todoText.trim()) {
+      dispatch(addTodo(todoText));
+      setTodoText('');
+    }
+  };
+
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTodoText(event.target.value);
+  };
+
+
+
+
+  return (
+    <Box
+      component='form'
+      onSubmit={handleSubmit}
+      sx={{ display: 'flex', alignItems: 'center', gap: 1, padding: '10px' }}
```

Comment:
Put them in the `makeStyles`?

Reply:
These styles have been moved to the `makeStyles`.